subsys/bluetooth/host/Kconfig

```diff
@@ -466,6 +466,43 @@ config BT_SMP_ALLOW_UNAUTH_OVERWRITE
 	  to create a new bond the old bond has to be explicitly deleted with
 	  bt_unpair.
 
+config BT_ID_UNPAIR_MATCHING_BONDS
```

Comment:
Should we s/MATCHING/CONFLICTING since the API you have below uses this language?

Reply:
As discussed oob, I'll keep it as is.